deps/wazuh_testing/wazuh_testing/modules/logcollector/event_monitor.py

```diff
@@ -0,0 +1,83 @@
+import re
+
+from wazuh_testing import T_30
+from wazuh_testing.modules.logcollector import LOG_COLLECTOR_PREFIX
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools import LOG_FILE_PATH
```

Comment:
Don't use global vars from tools. Add it to wazuh_testing init if needed and use it

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3484/commits/4db90e3c26ccd7561096df9d638a746b600ce208)